scene/gui/rich_text_label.cpp

```diff
@@ -4883,11 +4883,10 @@ void RichTextLabel::install_effect(const Variant effect) {
 	Ref<RichTextEffect> rteffect;
 	rteffect = effect;
 
-	if (rteffect.is_valid()) {
-		custom_effects.push_back(effect);
-		if ((!text.is_empty()) && use_bbcode) {
-			parse_bbcode(text);
-		}
+	ERR_FAIL_COND(!rteffect.is_valid());
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(rteffect.is_null(), "Invalid RichTextEffect resource.");
```
(or some similar message)
(the `is_null()` change isn't necessary, but it allows to skip `!`)